Formula/k3d.rb

```diff
@@ -1,3 +1,6 @@
+require "net/http"
```

Comment:
Can we do this just inside the `install` method?

Reply:
:heavy_check_mark: